src/index.ts

```diff
@@ -25,12 +26,12 @@ withAllStdIn((inputBuff: Buffer) => {
     const exportMap = new ExportMap();
     const fileNameToDescriptor: {[key: string]: FileDescriptorProto} = {};
 
-    // Generate separate `.ts` files for services if param is set
     const parameter = codeGenRequest.getParameter();
-    const generateGrpcWebServices = parameter === "service=grpc-web" || parameter === "service=true";
-    const generateGrpcNodeServices = parameter === "service=grpc-node";
+    const {service, mode} = parse(parameter, ",");
+    const generateGrpcWebServices = service === "grpc-web" || service === "true";
+    const generateGrpcNodeServices = service === "grpc-node";
 
-    if (parameter === "service=true") {
+    if (service === "true") {
```

Comment:
The `service` parameter should probably have more validation and helpful warnings, but this additional parameter makes that even clearer.

Can you add checks for the supported `service` and `mode` values and change this to throw exceptions if they're unrecognised to help users more quickly identify misconfigurations please?